### ISSUE TITLE
🐞 反馈问题：OCR 准确率有待提高

### ISSUE THREAD

**Issue by MoienBowen** [tisfeng/Easydict#146]
### 请先确认以下事项：

- [X] 已仔细阅读了 [README](https://github.com/tisfeng/Easydict#readme)
- [X] 在 [issues](https://github.com/tisfeng/Easydict/issues) 页面搜索过问题（包括已关闭的 issue），但未能找到解决方法
- [X] Easydict 已升级到最新版本

### 问题描述

使用 EasyDict 的 OCR 功能，经常会遇到英文、符号识别错误的问题（平常不会对中文进行 OCR）。

比如针对下图 PDF 进行 OCR，使用 EasyDict 和系统自带 OCR （捷径中相关功能）识别的结果如下。可以发现中括号、To 等有被识别错误的问题。

![Screenshot 2023-07-21 at 15 14 29](https://github.com/tisfeng/Easydict/assets/17046239/8b008cf7-bd27-43ed-93ab-bbe0c40f8479)

EasyDict：

Prior work [81, 1161, [57J has shown that an untrusted server can directly ex tract user-data from the model gradients. In these attacks, the server leverages its control over the shared model 2 In [16, the server exploits this ability to insert a fully-connected model layer as an extraction module where individual data points can be directly extracted. [8/ manipulates the model weights with an attack they call trap weights. This attack increases natural data leakage from fully-connected model layers. In [571, the server instead modifies model parameters to extract single data points by increasing their gradient contribution. The attack operates in several iterations of the protocol to collect multiple gradient updates from the same user.

The presence of such data extraction attacks highlights the vulnerability of vanilla FL to privacy-leakage. 1o prevent training data from leaking onto updates and straight

系统：

Prior work [8], [16], [57] has shown that an untrusted server can directly extract user-data from the model gradi-ents. In these attacks, the server leverages its control over the shared model? In [16], the server exploits this ability to insert a fully-connected model layer as an extraction module where individual data points can be directly ex-tracted. 8 manipulates the model weights with an attack they call trap weights. This attack increases natural data leakage from fully-connected model layers. In [57], the server instead modifies model parameters to extract single data points by increasing their gradient contribution. The attack operates in several iterations of the protocol to collect multiple gradient updates from the same user.
The presence of such data extraction attacks highlights the vulnerability of vanilla FL to privacy-leakage. To prevent training data from leaking onto updates and straight

### 重现步骤

如上。

### 期望结果

优化 OCR 结果。

### 是否愿意提交 PR 解决该问题？

- [ ] 我愿意提交 PR!

**Comment by tisfeng** [comment]
目前 OCR 识别准确率确实有待提高，但这个锅主要还得系统来背，这里 OCR 代码基本就是调用了系统的 API。

经多次测试，快捷指令的 OCR 准确率似乎是要好一点，但感觉和 Easydict 相比也并没有明显的提高。如果捷径也是使用的系统 OCR API，而不是某些私有 API 的话，那理论上效果应该是一样的。当然不排除系统有黑魔法，对捷径的 OCR 功能有特别的优化。

另外还有一点，对于同一个截图，如果图片区域有轻微的变化，OCR 的结果也可能不一样，需要多试几次。这是系统 OCR 的 feature 🤔

这个问题比较麻烦，暂时没想到优化方法，再看看。

<img width="456" alt="image" src="https://github.com/tisfeng/Easydict/assets/25194972/e991ac85-9e34-4cda-9036-540f14d12bd7">

<img width="1236" alt="image" src="https://github.com/tisfeng/Easydict/assets/25194972/3411294c-2fa9-4032-8073-1494fc59c3b3">
